RELEASE-NOTES.md

```diff
@@ -14,6 +14,7 @@
 * Empty strings are now detected as invalid language codes in the term classes
 * Made the `Item` constructor parameters optional
 * Made the `Fingerprint` constructor parameters optional
+* Made the `Property` constructor parameter for fingerprint nullable
```

Comment:
So you want me to revert to the wording you complained about? Sorry, but no. This is not a way forward. I still don't know what the problem was that made you changed it.